src/states/node.rs

```diff
@@ -940,7 +940,7 @@ impl Node {
             events.add_event(Event::NodeAdded(*name, self.peer_mgr.routing_table().clone()));
         }
         self.is_approved = true;
-
+        let _ = self.send_rt_request();
```

Comment:
if there is invariant failure in add_prefix, unrap on error will terminate immediately, won't reach this point